content/en/blog/_posts/2023-07-24-unknown-version-interoperability-proxy-alpha.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: blog
+title: "Kubernetes 1.28: Unknown Version Interoperability Proxy (Alpha)"
```

Comment:
I prefer: `Upgrade Kubernetes more safely by enabling the Unknown Version Interoperability Proxy (Alpha)`.

Reply:
Updated